test/helpers/index.js

```diff
@@ -90,6 +104,7 @@ function functionToString (func, opts = {}) {
 }
 
 function print (name, color, str) {
+  return
```

Comment:
meant to be here?

Reply:
yes, I would like to add some debug flag later if that's okay, all the printing can be removed later

Reply:
ok np